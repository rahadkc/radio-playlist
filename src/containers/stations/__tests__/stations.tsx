import { cleanup, render } from '@testing-library/react'
import Stations from '../index'

describe('<Stations>', () => {
  afterEach(cleanup)

  test('render Stations component', async () => {
    const element = render(<Stations />)
    expect(element).toMatchSnapshot()
  })
})
