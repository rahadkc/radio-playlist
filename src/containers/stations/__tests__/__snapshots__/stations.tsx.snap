// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stations> render Stations component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="wrapper undefined"
      >
        <header>
          <div
            class="container"
          >
            <div
              class="box"
            >
              <div
                class="grid"
              >
                <div
                  class="backButton"
                  style="background: url(/css_sprites.png) -94px;"
                >
                  &lt;
                </div>
                <div
                  class="title"
                >
                  STATIONS
                </div>
                <div
                  class="stopButton"
                  style="background: url(/css_sprites.png) -94px;"
                >
                  stop
                </div>
              </div>
            </div>
          </div>
        </header>
        <main
          class="main undefined"
        >
          <div
            class="container hFull"
          >
            <div
              class="box hFull"
            >
              <div
                class="loading"
              >
                Loading...
              </div>
            </div>
          </div>
        </main>
        <footer
          class="footer"
        >
          <div
            class="container"
          >
            <div
              class="box"
            />
          </div>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="wrapper undefined"
    >
      <header>
        <div
          class="container"
        >
          <div
            class="box"
          >
            <div
              class="grid"
            >
              <div
                class="backButton"
                style="background: url(/css_sprites.png) -94px;"
              >
                &lt;
              </div>
              <div
                class="title"
              >
                STATIONS
              </div>
              <div
                class="stopButton"
                style="background: url(/css_sprites.png) -94px;"
              >
                stop
              </div>
            </div>
          </div>
        </div>
      </header>
      <main
        class="main undefined"
      >
        <div
          class="container hFull"
        >
          <div
            class="box hFull"
          >
            <div
              class="loading"
            >
              Loading...
            </div>
          </div>
        </div>
      </main>
      <footer
        class="footer"
      >
        <div
          class="container"
        >
          <div
            class="box"
          />
        </div>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
